### DISCUSSION TITLE
Require some clarity around how the KafkaMirrorMaker2 resource is used with multiple mirror flows

### DISCUSSION THREAD

**Discussion by murphy-25** [strimzi/strimzi-kafka-operator#8325] [Q&A] [answered]
Having read the documentation I need a little clarity on the following, may have overlooked something. If I have two mirroring flows like so, with each flow mirroring different topic(s):
KafkaClusterA -> KafkaClusterB
KafkaClusterB -> KafkaClusterA

With that in mind do I need two KafkaMirrorMaker2 resources or can both flows be specified in the same resource in the list of mirrors?

Thanks M.

**Comment by scholzj** [comment, marked as answer]
You will need two mirror makers since one MM2 instance can have only one target cluster.
